fixbackend/dispatcher/collect_progress.py

```diff
@@ -56,4 +58,10 @@ def to_json_str(self) -> str:
 
     @staticmethod
     def from_json_str(value: bytes | str) -> "AccountCollectProgress":
+        # todo: remove the next day after this change is rolled out
+        dict_value = json.loads(value)
+        if dict_value.get("cloud") is None:
+            dict_value["cloud"] = CloudNames.AWS
```

Comment:
Why AWS?

Reply:
Just something to put here and avoid optional values. Won't be relevant in the next collect.